Enable int16 for op permute

3l1 opened the pull request:
Summary: Enable int16 for op permute

Reviewed By: Ninja91

Differential Revision: D84948536

3l1 commented:
@pytorchbot label "release notes: none"

digantdesai commented on backends/arm/test/ops/test_permute.py:
```diff
@@ -42,10 +41,10 @@ class SimplePermute(torch.nn.Module):
 
     def forward(self, x):
-        return self.permute(x)
+        return torch.permute(x, self.dims)
```
Add int16 test?

digantdesai reviewed (approved):
Review automatically exported from Phabricator review in Meta.

3l1 commented on backends/arm/test/ops/test_permute.py:
```diff
@@ -42,10 +41,10 @@ class SimplePermute(torch.nn.Module):
 
     def forward(self, x):
-        return self.permute(x)
+        return torch.permute(x, self.dims)
```
it is to avoid importing torchvision

3l1 committed:
Enable int16 for op permute (#15256)

Summary:

Enable int16 for op permute

bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

Reviewed By: Ninja91, digantdesai

Differential Revision: D84948536
